app/assets/stylesheets/components/_topic-list.scss

```diff
@@ -1,5 +1,5 @@
 .app-c-topic-list__item {
-  @include govuk-font(19, $weight: bold);
+  @include govuk-font(19, $weight: bold, $line-height: 1.7);
```

Comment:
should 1.7 be stored in a variable or does it not matter?

Reply:
Unless we use the value in multiple places, we only require variables when assigned directly as a property value – when passed as function arguments it's not required.